unity/Group55TestProjekt/Assets/Scripts/CellModel.cs

```diff
@@ -0,0 +1,253 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using MathNet.Numerics.Distributions;
+using MathNet.Numerics.Random;
+
+public class Cell
```

Comment:
Tried keeping the old name and it seems to work just fine. So what was the bug with the old name?

Reply:
It worked fine in the beginning, but then I got an error message that some quick googling suggested was due to non-monobehaviour scripts having identical class- and filenames. Might be fine, but thought this was safer